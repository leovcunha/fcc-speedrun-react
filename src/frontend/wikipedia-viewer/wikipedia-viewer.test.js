import React from "react";
import renderer from "react-test-renderer";
import WikipediaViewer from "../wikipedia-viewer";

describe("wikipedia-viewer", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<WikipediaViewer/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
