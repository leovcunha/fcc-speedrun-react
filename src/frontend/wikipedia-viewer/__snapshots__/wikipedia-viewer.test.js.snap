// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wikipedia-viewer renders without crashing 1`] = `
<div
  className="wikipedia-viewer jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Wikipedia Viewer
  </div>
  <hr
    className="p-5 undefined"
  />
  <form
    className="form-inline justify-content-center"
    onSubmit={[Function]}
  >
    <div
      className="form-group mx-sm-3 mb-5"
    >
      <input
        className="form-control"
        id="inputSearch"
        onChange={[Function]}
        placeholder="Search Wikipedia"
        type="search"
        value=""
      />
    </div>
    <button
      className="btn btn-secondary mb-5"
      type="submit"
    >
      Search
    </button>
    <button
      className="btn btn-secondary ml-2 mb-5"
      onClick={[Function]}
    >
      Random Article
    </button>
  </form>
  <div
    className="row"
  />
</div>
`;
